packages/webpack/src/plugin.js

```diff
@@ -74,6 +74,10 @@ class LavaMoatPlugin {
    */
   constructor(options = {}) {
     /** @type {LavaMoatPluginOptions} */
+    if (options.scuttleGlobalThis === true) {
+      options.scuttleGlobalThis = { enabled: true }
+    }
+    options.scuttleGlobalThis = Object.assign({}, options.scuttleGlobalThis)
```

Comment:
could this not be expressed as

```suggestion
    options.scuttleGlobalThis = {...options.scuttleGlobalThis}
```

?